File: src/main/java/com/extendedae_plus/helper/ExPatternPageAccessor.java
package com.extendedae_plus.helper;

/**
 * 由 GuiExPatternProviderMixin 实现，用于在客户端侧提供当前页号，避免反射读取 AE2 内部字段失败。
 */
public interface ExPatternPageAccessor {
    int eap$getCurrentPage();
}

File: src/main/kotlin/com/extendedae_plus/mixin/helper/ExPatternPageAccessor.kt
package com.extendedae_plus.mixin.helper

/**
 * 由 GuiExPatternProviderMixin 实现，用于在客户端侧提供当前页号，避免反射读取 AE2 内部字段失败。
 */
interface ExPatternPageAccessor {
    fun `eap$getCurrentPage`(): Int
}
